docs/Unity-Inference-Engine.md

```diff
@@ -9,6 +9,7 @@ to run the neural network within Unity.
 ## Supported devices
 
 Scripting Backends : The Unity Inference Engine is generally faster with __IL2CPP__ than with __Mono__ for Standalone builds.
+In the Editor, It is not possible to use the Unity Inference Engine with Graphics Emulation set to __OpenGL 3.0 emulation__.
```

Comment:
maybe it would be better like this:
"In the Editor, It is not possible to use the Unity Inference Engine with GPU device selected when Editor Graphics Emulation is set to __OpenGL(ES) 3.0 or 2.0 emulation__. Also there might be non-fatal build time errors when target platform includes Graphics API that does not support __Unity Compute Shaders__"